src/wp-includes/block-editor.php

```diff
@@ -191,13 +191,16 @@ function get_default_block_editor_settings() {
 
 	// These styles are used if the "no theme styles" options is triggered or on
 	// themes without their own editor styles.
-	$default_editor_styles_file = ABSPATH . WPINC . '/css/dist/block-editor/default-editor-styles.css';
-	if ( file_exists( $default_editor_styles_file ) ) {
+	$default_editor_styles_file                 = ABSPATH . WPINC . '/css/dist/block-editor/default-editor-styles.css';
+	static $default_editor_styles_file_contents = false;
+	if ( ! $default_editor_styles_file_contents && file_exists( $default_editor_styles_file ) ) {
```

Comment:
Does this change make any change? Instead of `false` use `true` than use the variable without not condition.

```suggestion
	static $default_editor_styles_file_contents = true;
	if ( $default_editor_styles_file_contents && file_exists( $default_editor_styles_file ) ) {
```

Reply:
Wouldn't that break the `if ( $default_editor_styles_file_contents ) {` condition a few lines below? 🤔 
We want `$default_editor_styles` to be an empty array - unless the file exists.
In order to improve performance, I set a `static $default_editor_styles_file_contents` variable to false initially. This way we can check if it's false, and only get the file-contents once. 
If we set it to true, then the `if ( $default_editor_styles_file_contents )` condition will always return true, so we'll be running the `file_get_contents` call again each time.
Am I missing something in the logic here? 🤔

Reply:
Make sense.